src/AppBundle/Resolver/SubSectionRoute/ProfCostsEstimateSubSectionRouteResolver.php

```diff
@@ -26,6 +27,10 @@ public function resolve(Report $report, $state)
             return self::SUMMARY_ROUTE;
         }
 
+        if ($this->managementCostSubsectionIsIncomplete($report, $state)) {
+            return self::MANAGEMENT_COST_ROUTE;
+        }
+
```

Comment:
shouldnt this be removed and the check go on the if statement below?

Reply:
Yep good spot - dropped it now.